resources/etc/cas/config/cas.properties.tpl

```diff
@@ -24,15 +24,57 @@ logging.config=file:/etc/cas/config/log4j2.xml
 # Configuration guide: https://apereo.github.io/cas/6.3.x/installation/LDAP-Authentication.html
 # Properties: https://apereo.github.io/cas/6.3.x/configuration/Configuration-Properties.html#ldap-authentication
 # ----------------------------------------------------------------------------------------------------------------------
-# Configure ldap-mapper as ldap source service
-cas.authn.ldap[0].ldap-url=ldap://ldap-mapper:3893/
-cas.authn.ldap[0].base-dn=ou=People,dc=cloudogu,dc=com
-cas.authn.ldap[0].type=DIRECT
-cas.authn.ldap[0].dn-format=uid=%s,ou=People,dc=cloudogu,dc=com
-cas.authn.ldap[0].search-filter=(&(objectClass=person)(uid={user}))
-cas.authn.ldap[0].principal-attribute-list=uid:username,cn,mail,givenName,sn:surname,displayName,memberOf:groups
```

Comment:
This line was necessary for attribute mapping, e.g. memberOf -> groups. Without this, the applications don't know about the users attributes. Try to log in as an admin. I think there will be no admin rights in the applications.

Reply:
Furthermore to support external user directories, this should be configurable like in CAS4: https://github.com/cloudogu/cas/blob/fc7a210826c81bf357303756958673aec20d422b/resources/startup.sh#L54...L61